tests/test_wallet.py

```diff
@@ -132,7 +146,9 @@ def test_init_default(self):
     def test_address(self):
         private_key = PrivateKey(WALLET_FORMAT_MAIN)
         assert private_key.address == BITCOIN_ADDRESS
-        private_key = PrivateKeyTestnet(WALLET_FORMAT_TEST)
+        private_key = PrivateKey(WALLET_FORMAT_TEST, network='test')
+        assert private_key.address == BITCOIN_ADDRESS_TEST
+        private_key = PrivateKey(WALLET_FORMAT_TEST, network='stn')
```

Comment:
Should we use a different `WALLET_FORMAT` and `BITCOIN_ADDRESS` for STN tests?

Reply:
Sure - that's no problem. Only reason I didn't was because it's technically all the same as far as WIFs are concerned. 
I will do this. I have family staying this week so on Saturday.

Reply:
I still think these should use different addresses. If just for aesthetics.

Reply:
🤣 Okay. Done. See latest commit. I actually did it initially and then changed it all back again because thought people might judge me and think I don't understand that it's all the same... Let's go with aesthetics 😉